src/runtime/composables/useFormField.ts

```diff
@@ -77,6 +77,7 @@ export function useFormField<T>(props?: Props<T>, opts?: { bind?: boolean, defer
     size: computed(() => props?.size ?? formField?.value.size),
     color: computed(() => formField?.value.error ? 'error' : props?.color),
     highlight: computed(() => formField?.value.error ? true : props?.highlight),
+    required: computed(() => formField?.value.required),
```

Comment:
Why not compute the `required` based on prop here instead of adding a new computed in each component? 🤔